fairing/builders/config_maps.py

```diff
@@ -0,0 +1,61 @@
+import os
+import subprocess
+import tempfile
+import shutil
+
+from kubernetes import client
+
+from fairing import notebook_helper
+from .builder import BuilderInterface
+from fairing.backend import kubernetes
+
+
+class ConfigMapBuilder(BuilderInterface):
+
+    def __init__(self, notebook_name=None):
+        if notebook_name is None:
+            self.notebook_name = notebook_helper.get_notebook_name()
```

Comment:
Does this work? If not, we should make notebook_name required.

Reply:
It works in standalone notebook, not with JupyterHub.
I can improve the error handling in the helper to display an explicit error when user JH such as: "notebook_name is required when using JupyterHub"